src/openzaak/components/zaken/api/viewsets.py

```diff
@@ -994,3 +1010,75 @@ def perform_destroy(self, instance):
             )
         super().perform_destroy(instance)
         return
+
+
+class ZaakContactMomentViewSet(
+    NotificationCreateMixin,
+    AuditTrailCreateMixin,
+    AuditTrailDestroyMixin,
+    ListFilterByAuthorizationsMixin,
+    CheckQueryParamsMixin,
+    mixins.CreateModelMixin,
+    mixins.DestroyModelMixin,
+    viewsets.ReadOnlyModelViewSet,
+):
+    """
+    Opvragen en bewerken van ZAAK-CONTACTMOMENT relaties.
+    list:
+    Alle ZAAKCONTACTMOMENTen opvragen.
+    Alle ZAAKCONTACTMOMENTen opvragen.
+    retrieve:
+    Een specifiek ZAAKCONTACTMOMENT opvragen.
+    Een specifiek ZAAKCONTACTMOMENT opvragen.
+    create:
+    Maak een ZAAKCONTACTMOMENT aan.
+    **Er wordt gevalideerd op**
+    - geldigheid URL naar de CONTACTMOMENT
+    destroy:
+    Verwijder een ZAAKCONTACTMOMENT.
+    """
```

Comment:
Please add some blank lines to keep this actually readable. The other viewsets do have this properly set up.